port/linux/storage.c

```diff
@@ -80,8 +81,10 @@ oc_storage_write(const char *store, uint8_t *buf, size_t size)
   if (!fp)
     return -EINVAL;
 
-  size = fwrite(buf, 1, size, fp);
+  size_t wsize = fwrite(buf, 1, size, fp);
+  fflush(fp);
+  fsync(fileno(fp));
```

Comment:
`fsync` is not available on Windows.

Reply:
But this is `port/linux` :)

Reply:
For windows we can use [_commit](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/commit?view=msvc-170)

Reply:
emulate [fsync](http://ab-initio.mit.edu/octave-Faddeeva/gnulib/lib/fsync.c)

Reply:
> But this is port/linux :)
Your're right - I somehow missed that...